themes/scss/widgets/themes/light.scss

```diff
@@ -217,42 +219,40 @@ $chartMiniHighlight: rgba(#333, 1);
     }
   }
 
-
   //Charts
   .extent {
     stroke: $link;
   }
 
-  .CDB-Chart-bar {
+  .CDB-Chart-bar--green {
```

Comment:
Isn't the green color the default color of a char bar? Is it necessary to have a modifier?